content/applications/general/export_import_data.rst

```diff
@@ -176,156 +228,171 @@ Example that will not work:
-have already been imported will be modified instead of being created. This is very useful as it
-allows you to import several times the same CSV file while having made some changes in between two
-imports. Odoo will take care of creating or modifying each record depending if it's new or not.
+Can the same record be imported several times?
+----------------------------------------------
 
-This feature allows you to use the Import/Export tool of Odoo to modify a batch of records in your
-favorite spreadsheet application.
+If a file is imported that contains one of the columns: `External ID` or `Database ID`, records that
+have already been imported will be modified instead of being created. This is extremely useful as it
+allows to import the same :abbr:`CSV (Comma-separated Values)` file several times while having made
+some changes in between two imports. Odoo will take care of creating or modifying each record
+depending if it is new or not.
 
-What happens if I do not provide a value for a specific field?
---------------------------------------------------------------
+This feature allows a company to use the Import/Export tool in Odoo to modify a batch of records in
+a spreadsheet application.
 
-If you do not set all fields in your CSV file, Odoo will assign the default value for every non
-defined fields. But if you set fields with empty values in your CSV file, Odoo will set the EMPTY
-value in the field, instead of assigning the default value.
+What happens if a value is not provided for a specific field?
+-------------------------------------------------------------
+
+If all fields are not set in the CSV file, Odoo will assign the default value for every non-defined
+fields. But if fields are set with empty values in the :abbr:`CSV (Comma-separated Values)` file,
+Odoo will set the EMPTY value in the field, instead of assigning the default value.
 
 How to export/import different tables from an SQL application to Odoo?
 ----------------------------------------------------------------------
 
-If you need to import data from different tables, you will have to recreate relations between
-records belonging to different tables. (e.g. if you import companies and persons, you will have to
-recreate the link between each person and the company they work for).
+If data needs to be imported from different tables, relations will need to be recreated between
+records belonging to different tables. (e.g. if companies and persons are imported, the link between
+each person and the company they work for will need to be recreated).
+
+To manage relations between tables, use the `External ID` facilities of Odoo. The `External ID` of a
+record is the unique identifier of this record in another application. The `External ID` must be
+unique across all records of all objects. It is a good practice to prefix this `External ID` with
+the name of the application or table. (like 'company_1', 'person_1' instead of '1')
```

Comment:
```suggestion
If data needs to be imported from different tables, relations need to be recreated between records
belonging to different tables. For instance, if companies and persons are imported, the link between
each person and the company they work for needs to be recreated.

To manage relations between tables, use the `External ID` facilities of Odoo. The `External ID` of a
record is the unique identifier of this record in another application. The `External ID` must be
unique across all records of all objects. It is a good practice to prefix this `External ID` with
the name of the application or table. (like, 'company_1', 'person_1' - instead of '1')
```